client/my-sites/pages/page-list.jsx

```diff
@@ -16,9 +16,8 @@ var PostListFetcher = require( 'components/post-list-fetcher' ),
 	NoResults = require( 'my-sites/no-results' ),
 	actions = require( 'lib/posts/actions' ),
 	Placeholder = require( './placeholder' ),
-	mapStatus = require( 'lib/route' ).mapPostStatus;
-
-import BlogPostsPage from './blog-posts-page';
+	mapStatus = require( 'lib/route' ).mapPostStatus,
+	sortPagesHierarchically = require( './helpers' ).sortPagesHierarchically;
```

Comment:
Thanks -- I'll clean this up in the feature branch (#13250) before we request review for merging it to master